### PR TITLE
Remove old files that are already transferred to s3

### PR DESCRIPTION
**Pull request by kcho**

### CODE REVIEW

File: lochness/mediaflux/__init__.py [hunk 1]
```diff
@@ -165,9 +173,9 @@ def sync_module(Lochness: 'lochness.config',
                             p = Popen(cmd, shell=True)
                             p.wait()
 
-                            # verify checksum after download completes
-                            # if checksum does not match, data will be downloaded again
-                            # ENH should we verify checksum 5 times?
+                            # verify checksum after download completes if
+                            # checksum does not match, data will be downloaded
+                            # again ENH should we verify checksum 5 times?
```
Comment (tashrifbillah) [inline lochness/mediaflux/__init__.py:178]: Do you conform to 80 characters manually or is there a library that automatically wraps lines at 80?

File: scripts/lochness_create_template.py [hunk 2]
```diff
@@ -305,6 +305,9 @@ def create_config_template(config_loc: Path, args: object) -> None:
 mindlamp_days_to_pull: 100
 pii_table: {args.pii_csv}
 lochness_sync_history_csv: {args.lochness_sync_history_csv}
+days_to_keep: 100
+removed_df_loc: {args.outdir}/PHOENIX/removed_files.csv
+removed_phoenix_root: {args.outdir}/track_removed_files_PHOENIX
```
Comment (tashrifbillah) [inline scripts/lochness_create_template.py:310]: Didn't Sylvain say 14 days for default?

File: lochness/mediaflux/__init__.py (hunk 1 above)
Comment (kcho) [reply lochness/mediaflux/__init__.py:178]: I'm using `set colorcolumn=79` in vim to highlight the PEP8 limit.

File: scripts/lochness_create_template.py (hunk 2 above)
Comment (kcho) [reply scripts/lochness_create_template.py:313]: Yes- about two weeks. I don't want to start deleting files yet, this is still at its testing phase.

**Comment by kcho** [comment]
@tashrifbillah I was going to remove you from the reviewer list, because this PR is not fully ready. I'll let you know when it's ready for your review

**Comment by kcho** [comment]
@tashrifbillah Requesting review-

This PR has quite a large update, which have been accumulating over the last couple of weeks.
The functions used in scanning through data sources are also included in this PR.

File: lochness/email/__init__.py [hunk 3]
```diff
@@ -195,16 +196,19 @@ def send_out_daily_updates(Lochness, days: int = 1,
 
         send_detail(
             Lochness,
+            Lochness['sender'],
+            Lochness['notify'],
```
Comment (tashrifbillah) [inline lochness/email/__init__.py:200]: Please double-check whether the above two `Lochness['sender']` should be just `sender` that you passed to this function newly.

File: lochness/utils/path_checker.py [hunk 4]
```diff
@@ -59,13 +59,17 @@ def update_interviews_transcript_check(df: pd.DataFrame) -> pd.DataFrame:
...
     transcript_int_df['subject_check'] = transcript_int_df['subject'
             ].str.match('[A-Z]{2}\d{5}').fillna(False)
 
     # check site and AMPSCZ IDs in the transcript file name
     for index, row in transcript_int_df.iterrows():
+        if not row['subject_check']:
+            transcript_int_df.loc[index, 'subject'] = re.search(
+                r'[A-Z]{2}\d{5}', row['subject']).group(0)
+            row['subject'] = transcript_int_df.loc[index, 'subject']
+
```
Comment (tashrifbillah) [inline lochness/utils/path_checker.py:72]: The `re.search` stuff are hard to verify from reviewer's perspective. Please test adequately using Python debugger.

File: lochness/email/__init__.py (hunk 3 above)
Comment (tashrifbillah) [reply lochness/email/__init__.py:200]: Never mind!

File: lochness/utils/source_check.py [hunk 5]
```diff
@@ -216,23 +216,26 @@ def send_source_qc_summary(qc_fail_df: pd.DataFrame,
...
         'do not follow the SOP. Please move, rename or delete the files '
         'according to the SOP. Please do not hesitate to get back to us. '
-        '<br><br>Best wishes,<br>DPACC',
+        '<br><br>Best wishes,<br>DPACC<br><br><br>',
         table_str,
         lines,
-        yPlease let us know if any of the files above '
-        'should have passed QC')
+        'Please let us know if any of the files above should have passed QC'
+        )
```
Comment (tashrifbillah) [inline lochness/utils/source_check.py:230]: Not sure about the use of `should have`. Couldn't it be just `... above pass QC'` ?

File: lochness/utils/source_check.py [hunk 6]
```diff
@@ -269,6 +272,7 @@ def check_source(Lochness: 'lochness') -> None:
         db_string = 'RPMS'
         mediaflux_df = collect_mediaflux_files_info(Lochness)
         all_df = check_file_path_df(mediaflux_df, subject_id_list)
+        all_df = all_df[all_df['site'].str.startswith('Prescient')]
```
Comment (tashrifbillah) [inline lochness/utils/source_check.py:275]: Are we reducing the `all_df` for speed purpose down the line?

Comment (kcho) [reply lochness/utils/source_check.py:275]: There are other directories in mediaflux, which are not being used in the Prescient study. These directories are being skipped.

File: lochness/utils/source_check.py (hunk 5 above)
Comment (kcho) [reply lochness/utils/source_check.py:230]: I'm asking people in the email to contact me back if the table includes any files that should have not been detected as "incorrect" in our algorithm.

**Comment by kcho** [comment]
The updates from this branch needs to be tested on the pronet production server. I'm merging this branch into master. I'm happy to get further comments on this branch, so I'll not delete the branch.

**Comment by tashrifbillah** [comment]
> I'm merging this branch into master.

Okay thank you. I have been busy with many other things so I wouldn't have gotten time to review it again anytime soon.

**Pull request merged**